macOS/DuckDuckGo/Application/AppDelegate.swift

```diff
@@ -388,6 +389,12 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
         privacyStats = PrivacyStats(databaseProvider: PrivacyStatsDatabase())
 #endif
         PixelKit.configureExperimentKit(featureFlagger: featureFlagger, eventTracker: ExperimentEventTracker(store: UserDefaults.appConfiguration))
+
+#if DEBUG
+        faviconManager = FaviconManager(cacheType: AppVersion.runType == .normal ? .standard : .inMemory)
```

Comment:
should we check for `requiresEnvironment` instead to cover Int/UI Tests as well or `.normal`+`.uiTests`+`.uiTestsOnboarding` here?

Reply:
Great point, let me update to check for `requiresEnvironment` 👍